src/aura/STG_CMP_Addr/STG_CMP_Addr.cmp

```diff
@@ -89,6 +89,32 @@
             <ui:outputText value="{!$Label.c.stgHelpContactPreferredPhone}" class="slds-text-body--small" />
         </div>
 
+        <div class="slds-col slds-size--1-of-2">
```

Comment:
I put this below "Disable Preferred Phone enforcement" but I actually think it should go above it and put a comment out to Julie/Sara to see what they say. 

If this sync is disabled, then the "Disabled Preferred Phone enforcement" setting doesn't even matter.

Reply:
Ideally maybe we could enhance the UI further to not even let the user touch the "Disabled Preferred Phone enforcement" setting if it's overall disabled but I'm not great at UI and not sure how much time we want to invest in this.

Reply:
@robertbwatson  Thanks for all the hard work in fixing this.  Do we want to disable the "Run Clean Up" button when "Disable Preferred Email" is Checked and "Enable Preferred Phone Sync" is unchecked, may be in the future?

Reply:
@skosgi1  Good question. I thought about it but didn't want to change anything here. We might want to create a followup WI for it though. Let's discuss more tomorrow.

Reply:
There are scores of issues like this in settings that need to be addressed. However I think it lies out of scope of the issue at hand.

Reply:
Created W-036335 to discuss the Run Cleanup button in a future release.